microsoft-edge/webview2/how-to/debug-visual-studio-code.md

```diff
@@ -51,7 +52,7 @@ Visual Studio Code source path mapping now requires an URL, so your app now rece
 
 1. Click **Start Debugging**, which is the green triangle next to the launch configuration dropdown.
 
-   ![The Run tab in Visual Studio Code.](./media/run-vs.png)
+   ![The Run tab in Visual Studio Code.](./media/run-vscode.png)
```

Comment:
```suggestion
   ![The Run tab in Visual Studio Code](./media/run-vscode.png)
```
moving away from period in alt text for simplicity and common practice, might be in wiki > Edge Dev Style Guide.  It's clear we'll need to do a global PR to stop them propagating.